### PR TITLE
[9.0] add available resource category tab on activities

### PR DESCRIPTION
**Pull request by robinkeunen**
[T0167 - A13 - Activité : voir les vélos disponibles par catégories](https://gestion.coopiteasy.be/web#id=863&view_type=form&model=project.task&action=479)

- add resource planning basic demo data
- get resource allocations for time window and location
- get categories available and resource count for time window and location
- display available categories on activity form
  - the field is computed. I needed to store it to display it properly (otherwise, it disappeared out of edit mode and was only computed if a dependent field was touched)
  - the discarded resource.category.available records are garbage collected daily

### CODE REVIEW

File: resource_activity/__openerp__.py [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "reports/layouts.xml",
         "views/menus.xml",
     ],
+    "demo": ["demo/demo.xml",],
```
Comment (houssine78) [inline resource_activity/__openerp__.py:63]: bump version ;)

File: resource_planning/__openerp__.py [hunk 2]
```diff
@@ -43,6 +43,7 @@
         "views/actions.xml",
         "views/menus.xml",
     ],
+    "demo": ["demo/demo.xml",],
```
Comment (houssine78) [inline resource_planning/__openerp__.py:46]: bump version

**Review by houssine78 — APPROVED** [review]

File: resource_activity/__openerp__.py (hunk 1 above)
Comment (robinkeunen) [reply resource_activity/__openerp__.py:63]: ça va finir par rentrer!

Comment (houssine78) [reply resource_activity/__openerp__.py:63]: sinon faire la demande à l'équipe infra pour avoir notre bot coopiteasy ;)

Comment (robinkeunen) [reply resource_activity/__openerp__.py:63]: Ou, utiliser celui que le GRAP met grâcieusement à notre disposition ;-)

**Pull request merged**